File: uri/media/media.pas
program foo(Input, Output);

uses
    SysUtils;

function media( notaA: Double; notaB: Double) : Double;
var
    pesoA : Double;
    pesoB : Double;

begin
    pesoA := 3.5;
    pesoB := 7.5;
    media := ((notaA * pesoA) + (notaB * pesoB)) / (pesoA + pesoB);
end;

var 
    notaA : Double;
    notaB : Double;
    fullmedia : Double;    

begin
    readLn(notaA);
    readLn(notaB);
    fullmedia := media(notaA, notaB);    
    writeLn('MEDIA = ', Format('%.5f', [fullmedia]) );
end.

File: uri/media/media.lua
function media( notaA , notaB )
    return (((notaA * 3.5) + (notaB * 7.5)) / 11.0 )
end

local notaA, notaB, res
notaA = io.read("*n")
notaB = io.read("*n")
res = string.format("%.5f", media( notaA, notaB ))
print("MEDIA = "..res)
